### ISSUE TITLE
PSR12.Files.FileHeader detects @var annotations as file docblocks

### ISSUE THREAD

**Issue by omsi668** [squizlabs/PHP_CodeSniffer#2626]
PHPCS Version : 3.5.0
PHP Version : 7.2.19

Hello,

On my project, I've got a lot of view files with no File Header but with a first DocBlock declaring vars to avoid bad inspection and use autocomplete.

These files raise an error with the new standard : 
 5 | ERROR | The file-level docblock must follow the opening PHP tag in the file header
   |       | (PSR12.Files.FileHeader.IncorrectOrder)

Here is the start of my file : 
```
<?php

use common\models\JobFunction;

/** @var JobFunction $model */
```

As you can see, this is not a file level docblock and therefore it should not raise this error.

Thank you, have a good day

**Comment by gsherwood** [comment]
Thanks for the report. I've fixed this issue in master and it will be released in 3.5.1
